### PR TITLE
perf(native-filters): Load native filters after charts

### PR DESCRIPTION
**Pull request by simcha90**
### SUMMARY
Show filter bar after all charts loaded

Dependent on: https://github.com/apache/superset/pull/14408

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/56388545/116966766-9691d900-acb9-11eb-8115-3361ecc87792.mov

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by villebro** [comment]
Reviewing/testing #14408 first, but one problem with this approach is that it will cause two renders of charts - first with the full width of the dashboard without the filter tab, and next with the filter tab. To address this I would recommend making the filter tab visible initially, but potentially introduce the loading spinner instead of the tab contents to highlight that filters are still pending.

File: superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx [hunk 1]
```diff
@@ -238,7 +243,9 @@ const FilterBar: React.FC<FiltersBarProps> = ({
           dataMaskSelected={dataMaskSelected}
           dataMaskApplied={dataMaskApplied}
         />
-        {isFeatureEnabled(FeatureFlag.DASHBOARD_NATIVE_FILTERS_SET) ? (
+        {!isInitialized ? (
+          <Loading />
+        ) : isFeatureEnabled(FeatureFlag.DASHBOARD_NATIVE_FILTERS_SET) ? (
```
Comment (villebro) [inline superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx:248]: I think this causes an unnecessary visual artefact - I think it's ok to keep this as-is, i.e. showing the filters in spinning state until they've fully resolved.

**Review by villebro — COMMENTED** [review]
One small comment, other than that LGTM and was verified to work well in all following cases:
- regular examples dashboard
- dashboard with no charts on first page
- Global Async Query framework (websocket)

File: superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx (hunk 1 above)
Comment (simcha90) [reply superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx:248]: but this is the point to render them in order they will not start to do requests to the server :)

Comment (villebro) [reply superset-frontend/src/dashboard/components/nativeFilters/FilterBar/index.tsx:248]: Haha, makes sense 🤦

**Review by villebro — APPROVED** [review]
LGTM

**Pull request merged**